src/ruby/ext/grpc/extconf.rb

```diff
@@ -85,6 +86,7 @@ def env_append(name, string)
 
 env_append 'CPPFLAGS', '-DGPR_BACKWARDS_COMPATIBILITY_MODE'
 env_append 'CPPFLAGS', '-DGRPC_XDS_USER_AGENT_NAME_SUFFIX="\"RUBY\""'
+env_append 'CPPFLAGS', '-DGPR_WINDOWS_UCRT' if windows_ucrt
```

Comment:
minor naming nit: this isn't a typical "GPR" macro, like the rest of the ones defined [here](https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/port_platform.h) for the C-core library, so it feels out of place to use that here.

Can we rename this to `GRPC_RUBY_WINDOWS_UCRT` ?

Reply:
@apolcyn done